fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -168,7 +168,7 @@ public CommandProcessingResult submitApplication(final JsonCommand command) {
 
                     Group group = this.groupRepositoryWrapper.findOneWithNotFoundDetection(groupId);
 
-                    if (command.booleanObjectValueOfParameterNamed("isParentAccount") != null) {
+                    if (command.booleanObjectValueOfParameterNamed("isParentAccount")) {
```

Comment:
How does this fix anything?

Reply:
It is a boolean value that is being returned by `command.booleanObjectValueOfParameterNamed("isParentAccount")`. It will always evaluate to `true` when compared to not be null.